templates/foreman.service-overrides.erb

```diff
@@ -4,4 +4,4 @@ Environment=FOREMAN_ENV=<%= scope['foreman::rails_env'] %>
 Environment=FOREMAN_HOME=<%= scope['foreman::app_root'] %>
 Environment=FOREMAN_PUMA_THREADS_MIN=<%= scope['foreman::config::min_puma_threads'] %>
 Environment=FOREMAN_PUMA_THREADS_MAX=<%= scope['foreman::foreman_service_puma_threads_max'] %>
-Environment=FOREMAN_PUMA_WORKERS=<%= scope['foreman::foreman_service_puma_workers'] %>
+Environment=FOREMAN_PUMA_WORKERS=<%= scope['foreman::config::puma_workers'] %>
```

Comment:
This is generated from the `foreman::config` scope so I think this is equal:
```suggestion
Environment=FOREMAN_PUMA_WORKERS=<%= @puma_workers %>
```

The same is true for `min_puma_threads`.